src/Exception/EPoetryException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Exception;
+
+/**
+ * Exception thrown when issues exist when using ePoetry.
+ */
+class EPoetryException extends \Exception
```

Comment:
The `EPoetry` bit is already in the namespace, let's call this `ClientException` for now.

Reply:
Done.